addons/mail/static/src/models/attachment_box_view.js

```diff
@@ -33,6 +33,13 @@ registerModel({
             }
             return clear();
         },
+        /**
+         * @private
+         * @returns {Boolean}
+         */
+        _computeIsUploadDisabled() {
+            return Boolean(this.chatter.thread.hasWriteAccess);
```

Comment:
If `hasWriteAccess` is a boolean, no need to wrap into Boolean.

Reply:
Done